From 055e12414a27fbf60e083e1d40e13f17d68d928d Mon Sep 17 00:00:00 2001
From: Hongxu Jia <hongxu.jia@windriver.com>
Date: Tue, 17 Aug 2021 16:00:24 +0800
Subject: [PATCH] fix build //tensorflow/lite/examples/label_image:label_image
 failure

It caused undefined reference neon_tensor_utils failure
...
|tensor_utils.cc:(.text._ZN6tflite12tensor_utils18ReductionSumVectorEPKfPfii+0x0):
undefined reference to
`tflite::tensor_utils::NeonReductionSumVector(float const*,
float*, int, int)'
...

Define Yocto specific cpu `armeabi' for lite label_image
Upstream-Status: Inappropriate [oe specific]

Signed-off-by: Hongxu Jia <hongxu.jia@windriver.com>
---
 tensorflow/lite/kernels/internal/BUILD | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/tensorflow/lite/kernels/internal/BUILD b/tensorflow/lite/kernels/internal/BUILD
index 11292aeb2bb..1746336cd1d 100644
--- a/tensorflow/lite/kernels/internal/BUILD
+++ b/tensorflow/lite/kernels/internal/BUILD
@@ -135,6 +135,13 @@ config_setting(
     },
 )
 
+config_setting(
+    name = "armeabi",
+    values = {
+        "cpu": "armeabi",
+    },
+)
+
 config_setting(
     name = "haswell",
     values = {
@@ -273,6 +280,7 @@ selects.config_setting_group(
         ":arm",
         ":arm64-v8a",
         ":armeabi-v7a",
+        ":armeabi",
         ":armhf",
         ":armv7a",
         ":ios_armv7",
